config/config.go

```diff
@@ -892,6 +892,8 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.madvertise.endpoint", "https://mobile.mng-ads.com/bidrequest{{.ZoneID}}")
 	v.SetDefault("adapters.marsmedia.endpoint", "https://bid306.rtbsrv.com/bidder/?bid=f3xtet")
 	v.SetDefault("adapters.mediafuse.endpoint", "http://ghb.hbmp.mediafuse.com/pbs/ortb")
+	v.SetDefault("adapters.medianet.endpoint", "https://prebid-adapter.media.net/rtb/pb/prebids2s")
+	v.SetDefault("adapters.medianet.extra_info", v.GetString("external_url"))
```

Comment:
What's the purpose of this function call? Are we trying to set the Medianet adapter's `config.ExtraAdapterInfo` to `"http://localhost:8000"`? Is this a local test leftover code? Assuming these default values are used, your adapter's endpoint would be: `"https://prebid-adapter.media.net/rtb/pb/prebids2s?src=http%3A%2F%2Flocalhost%3A8000"`.

Reply:
We are passing  external_url as a param to request in MediaNet endpoint for a feature that MediaNet has implemented at their end. This is not a local test leftover code. default values are for local env but when we will use any config file (pbs.yaml or pbs.json) then in that case external_url will be overridden in viper.

Reply:
At the runtime it will be PBS prod url, not localhost.

Reply:
Prebid Server will officially support this feature very soon, as discussed in issue #1739 which is still open and pending to be coded. As of right now, can you please either:
 1. Hardcode the Host URL as the default value of `external_url` field
```
894     v.SetDefault("adapters.mediafuse.endpoint", "http://ghb.hbmp.mediafuse.com/pbs/ortb")
895     v.SetDefault("adapters.medianet.endpoint", "https://prebid-adapter.media.net/rtb/pb/prebids2s")
896 -   v.SetDefault("adapters.medianet.extra_info", v.GetString("external_url"))
    +   v.SetDefault("adapters.medianet.extra_info", "https://medianets.default.host.com")
897     v.SetDefault("adapters.mgid.endpoint", "https://prebid.mgid.com/prebid/")
898     v.SetDefault("adapters.mobilefuse.endpoint", "http://mfx.mobilefuse.com/openrtb?pub_id={{.PublisherID}}")
899     v.SetDefault("adapters.mobfoxpb.endpoint", "http://bes.mobfox.com/?c=__route__&m=__method__&key=__key__")
config/config.go
```
 2. Or delete this default value entry entirely and just make sure implementations of prebid server are configured with this field. If so, make your adapter disabled by default because check `if err != nil || len(hostUrl) == 0` in line 111 of `adapters/medianet/medianet.go` discards empty host url strings and errors out upon Prebid Server start if this value is not found.
```
894     v.SetDefault("adapters.mediafuse.endpoint", "http://ghb.hbmp.mediafuse.com/pbs/ortb")
895     v.SetDefault("adapters.medianet.endpoint", "https://prebid-adapter.media.net/rtb/pb/prebids2s")
896 -   v.SetDefault("adapters.medianet.extra_info", v.GetString("external_url"))
    +   v.SetDefault("adapters.medianet.disabled", true)
897     v.SetDefault("adapters.mgid.endpoint", "https://prebid.mgid.com/prebid/")
898     v.SetDefault("adapters.mobilefuse.endpoint", "http://mfx.mobilefuse.com/openrtb?pub_id={{.PublisherID}}")
899     v.SetDefault("adapters.mobfoxpb.endpoint", "http://bes.mobfox.com/?c=__route__&m=__method__&key=__key__")
config/config.go
```